tests/context.py

```diff
@@ -24,61 +26,179 @@ def _Pow(a, b):
 
 def get_simple_examples(func):
     '''
-    Returns an array of tuples, containing the string `input`, sympy `output` using the provided sympy `func`, and `symbolically` boolean
```

Comment:
@builtbywill I can dig around and try to figure this out, but why do we not need the `symbolically` boolean for calling `compare` anymore?